src/index.js

```diff
@@ -1,48 +1,103 @@
 import { handleDragStart, handleDrop, cancelDefault } from './services/sort';
-import { updateStatus, getTodos } from './services/update';
+import {
+  updateStatus, getTodos, saveToStorage, orderAndSave,
+} from './services/update';
+import {
+  createTodo, clearAll, clearList, editTodoInStorage,
+  editTodoInList, deleteTodoFromStorage, deleteTodoFromList,
+} from './services/todoService';
 import './style.css';
 
 class TodoList {
   constructor() {
     this.todos = getTodos();
   }
 
-  handleCompletedStatus = (e) => {
-    updateStatus(e, this.todos);
-  }
+ handleCompletedStatus = (e) => {
+   const todos = getTodos();
+   updateStatus(e, todos);
+ }
 
-  handleDropEvt = (e) => {
-    handleDrop(e, this.todos);
-  }
+ handleEditTodo = (e) => {
+   if (e.keyCode === 13) {
+     const { value } = e.target;
+     const index = parseInt(e.target.dataset.index, 10) - 1;
+     editTodoInStorage(value, index);
+     editTodoInList(value, index);
+   }
+ }
+
```

Comment:
- [ ] OPTIONAL: This method or maybe the called functions has bugs. When I edit the description, it does not update the local storage. As a result, when I refresh the page the description gets back to its original value. This should have been a mandatory change but because the previous reviewer did not point it out, I will leave it as an optional change. However, I strongly suggest you fix this issue. :pray: 

![Screenshot from 2021-09-24 19-43-59](https://user-images.githubusercontent.com/64894927/134685381-0d418ee0-7e62-4b5a-9a65-74897cff6dfa.png)

Reply:
I will look into that , thanks for the reviewer message